react/src/components/PayButton/PayButton.tsx

```diff
@@ -9,15 +9,15 @@ import { currencyObject, getCurrencyObject } from '../../util/satoshis';
 import BigNumber from 'bignumber.js';
 
 export interface PayButtonProps extends ButtonProps {
-  to: string;
+  to?: string | undefined;
```

Comment:
This one should not be optional. It's the address the payment will go to, so every paybutton/widget will always have one.